canonical-web-and-design/maas.io
# Remove search

At the moment, there is no search form implemented, and the `/docs/search` URL route doesn't seem to be hit (404).

But also, I'm not sure /docs/search is a good idea and needs more consideration. I think it's confusing for `/docs` to have search when the site itself doesn't.

If we want search for docs, I would think we should just add a site-wide search, which would then automatically search docs - since maas.io doesn't have many pages apart from docs. If we really wanted a *separate* search for docs, then I think it should be a "tab" or something within the central search experience.

(This decision might be different in the interim for snapcraft.io, which has a significant amount of non-docs content, and already has a pattern for docs search at docs.snapcraft.io. But in the long run I think search should be integrated into the central search experience even there.)

But I also think it would be reasonable to treat this as out of scope for simply getting `/docs` live - we can do it quickly afterwards.

QA
--

`./run`, go to the homepage and `/docs`, check they work. Check `/docs/search` is a 404, as it always was.
